pkg/odo/genericclioptions/context.go

```diff
@@ -72,6 +72,59 @@ func checkProjectCreateOrDeleteOnlyOnInvalidNamespace(command *cobra.Command, er
 	}
 }
 
+// getFirstChildOfCommand gets the first child command of the root command of command
+func getFirstChildOfCommand(command *cobra.Command) *cobra.Command {
+	// If command does not have a parent no point checking
+	if command.HasParent() {
+		// Get the root command and set current command and its parent
+		r := command.Root()
+		p := command.Parent()
+		c := command
+		for {
+			// if parent is root, then we have our first child in c
+			if p == r {
+				return c
+			}
+			// Traverse backwards making current command as the parent and parent as the grandparent
+			c = p
+			p = c.Parent()
+		}
+	}
+	return nil
+}
+
+func checkValidConfig(command *cobra.Command, configFileName string) {
+	_, err := config.NewLocalConfigInfo(configFileName, true)
+	// List of allowed firstchild command
```

Comment:
err ignored?

Reply:
Not really. It gets caught as the config is read again later with 2nd parameter set to false. This is specifically meant to fail if file is not found

Reply:
Handled